tests/test_removeperson.bats

```diff
@@ -57,15 +57,15 @@ function teardown {
 }
 
 
-@test "run 'killperson' with multiple arguments" {
+@test "run 'removeperson' with multiple arguments" {
```

Comment:
Please, add a single test that `killperson` is still working, we don't want to break it just yet.

Reply:
Added in b186d6c8deed4290b77b1117e61c80cbcbf44e84.